src/ui/layout/constraints.md

```diff
@@ -2003,16 +2052,16 @@ BoxConstraints.tight(Size size)
 ```
 
 If you revisit [Example 2](#example-2) above,
-it tells us that the screen forces the red
-`Container` to be exactly the same size as the screen.
-The screen does that, of course, by passing tight
+the screen forces the red `Container` to be
+exactly the same size as the screen.
+The screen achieves that, of course, by passing tight
 constraints to the `Container`.
 
-A _loose_ constraint, on the other hand,
-sets the **maximum** width and height, but lets the widget
-be as small as it wants. In other words,
-a loose constraint has a **minimum** width and height
-both equal to **zero**:
+Some boxes _loosen_ the constraints,
+meaning the maximum is maintained but the
+minimum is removed, so the widget can have
+a **minimum** width and height both equal to **zero**.
+For example, [`Center`][]:
```

Comment:
The code blow doesn't really have much to do with `Center`. It is just the definition of loose constraints. Yes, internally, `Center` loosens the incoming constraints, but I don't think the "For example, Center" makes that relationship clear.